deploy/app/modules/indexer/elasticcache.tf

```diff
@@ -10,11 +10,10 @@ module "caches" {
 
   app = var.app
   environment = terraform.workspace
-  
-  cache_limits = {
-    data_storage_GB = terraform.workspace == "prod" ? 20 : 1
-    ecpu_per_second = terraform.workspace == "prod" ? 10000 : 1000
-  }
+
+  # cache.m7g.large => 2 vCPUs, 6.38 GiB memory, 12.5 Gigabit network, $0.1264/hour
+  # cache.t4g.micro => 2 vCPUs, 0.5 GiB memory, 5 Gigabit network, $0.0128/hour
+  node_type = terraform.workspace == "prod" ? "cache.m7g.large" : "cache.t4g.micro"
```

Comment:
Just noting this is a lot less than 20GB...

Reply:
according to our current metrics, it looks like we are able to store more than 2M items with the 20GB. Hannah mentioned that 1M items should probably cover 50% of requests, so I wanted to try with something smaller to see how it goes.

Reply:
ok, let's get this merged and we can iterate afterwards. I'll change the instance type, though "practically same price" is ~40% more :P